app/views/docs/functions.phtml

```diff
@@ -683,6 +683,37 @@ func main() async throws {
     </li>
 </ul>
 
+<h2><a href="/docs/functions#scheduled-execution" id="scheduled-execution">Scheduled Execution</a></h2>
+<p>Appwrite supports scheduled function executions. You can schedule execution using <a href="https://en.wikipedia.org/wiki/Cron" target="_blank" rel="noopener">cron expressions</a> in the settings of your function. Cron supports recurring executions as frequently as <b>every minute</b>.</p>
```

Comment:
It would be nice to have some better documentation than that wiki because that wiki is more about cron than the cron expression syntax

Reply:
Let's update it post the current release as we are running short on time.

Reply:
@stnguyen90 Write smthing to start and send it to me on discord. I'll work this into another iteration.